redash/handlers/queries.py

```diff
@@ -451,7 +451,7 @@ def post(self, query_id):
         require_access(query, self.current_user, not_view_only)
 
         parameter_values = collect_parameters_from_request(request.args)
-        parameterized_query = ParameterizedQuery(query.query_text)
+        parameterized_query = ParameterizedQuery(query.query_text, org=self.current_org)
```

Comment:
Why aren't we using query.paremterized_query here?

Reply:
Using `query.parameterized` would create a `ParameterizedQuery` with the query schema baked into it which would, in turn, force parameter schema validation.

It might be desirable to verify the schema here as well, I just didn't want to change any behavior in that part of the code as part of this PR. Let me know if you think it's safe.

Reply:
Got it. Keep as is.